pandas/core/base.py

```diff
@@ -554,7 +560,19 @@ def is_any_frame():
             return result, True
         elif is_list_like(arg):
             # we require a list, but not an 'str'
-            return self._aggregate_multiple_funcs(arg, _level=_level, _axis=_axis), None
+            try:
+                return (
+                    self._aggregate_multiple_funcs(arg, _level=_level, _axis=_axis),
+                    None,
+                )
+
+            # if no results
```

Comment:
can you give a more comprehensive comment here.

Why are we raising ValueError (and not AttributeError from the inner function)?

Reply:
I guess AttributeError would be a better option if not inconvenient since ValueErrors are conflicting. Will change it